content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -176,7 +176,7 @@ kind of information you should provide.
 
       Like other `istioctl` tools, we generally recommend using a downloaded version that matches the version deployed in your cluster.
 
-      For the time being, analysis is generally backwards compatible, so that you can, for example, run the 1.9 version of `istioctl analyze` against a cluster running Istio 1.6 and expect to get useful feedback. Analysis rules that are not meaningful with an older Istio release will be skipped.
+      For the time being, analysis is generally backwards compatible, so that you can, for example, run the 1.10 version of `istioctl analyze` against a cluster running Istio 1.6 and expect to get useful feedback. Analysis rules that are not meaningful with an older Istio release will be skipped.
```

Comment:
```suggestion
      For the time being, analysis is generally backwards compatible, so that you can, for example, run the {{< istio_version >}} version of `istioctl analyze` against a cluster running an older Istio 1.x version and expect to get useful feedback. Analysis rules that are not meaningful with an older Istio release will be skipped.
```